### DISCUSSION TITLE
Mirrored image for External Camera

### DISCUSSION THREAD

**Discussion by fericitu** [capawesome-team/capacitor-mlkit#166] [Q&A]
First, thank you for the amazing work developing and supporting this library. Your efforts are incredibly helpful and much appreciated.

We are using an external camera for scanning barcodes in our project. To achieve this, we utilized a workaround by incorporating another library.

Currently, we are facing an issue where the camera feed is not mirrored. Is there a way to achieve that with the current setup?
Any guidance would go a long way, Thank you!

**Comment by robingenz** [comment]
Unfortunately, I have no experience with external cameras and no external camera at home to test. Maybe someone from the community has an idea.

**Reply by fericitu** [reply]
Thank you nonetheless!
